backend/src/main/java/gov/cdc/usds/simplereport/config/WebConfiguration.java

```diff
@@ -24,6 +24,8 @@ public class WebConfiguration implements WebMvcConfigurer {
   public static final String PATIENT_UPLOAD = "/upload/patients";
   public static final String RESULT_UPLOAD = "/upload/results";
   public static final String CONDITION_AGNOSTIC_RESULT_UPLOAD = "/upload/condition-agnostic";
+
```

Comment:
formatting nit - extra newline?

Reply:
Ahhh thank you! I'll update that in the part 3 PR 😅

Reply:
ooo oops -- I'll remove this in my next PR 😅 Thank you for catching this 🎣